exemples/tls-auth/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04
```

Comment:
what's the need ?

Reply:
for the version or the dockerfile ?

Dockerfile is used in docker-compose to create certificates, it's kinda the purpose of tls-auth example